cyclonedx_py/parser/pipenv.py

```diff
@@ -34,6 +34,7 @@ class PipEnvParser(BaseParser):
 
     def __init__(
             self, pipenv_contents: str, use_purl_bom_ref: bool = False,
+            omit_category: Set[str] = set(),  # noqa: B006
```

Comment:
add this non-essential/optional argument after the `*`

Reply:
Collides with not having `noqa: B006`.

Reply:
Or make `omit_category` `Optional[Set[str]]` ?

Reply:
i want this to be _after_ the `*` 
it is non-essential and must be called as a named parameter, therefore.

The idea to have this an optional set is implicit, as it would fix `bug B006`.
I hoped that @tngraf would find the solution.

Reply:
fixed